pyEpiabm/pyEpiabm/property/household_foi.py

```diff
@@ -31,7 +31,11 @@ def household_inf(infector, time: float):
             Infectiousness parameter of household
 
         """
-        return PersonalInfection.person_inf(infector, time)
+        household_infectiousness = PersonalInfection.person_inf(infector, time)
```

Comment:
Would be better to be consistent with how this is done in the function household_foi in this script. I see that it is also done like this in spatial_inf. Do we want to be consistent in all or is it okay to do it like this in the _inf functions?

And is it correct that we do it in this function instead of in household_foi?

Reply:
I think according to CovidSim 'Unlike other measures, place closure does not affect infectiousness and susceptibility functions in CalcInfSusc.cpp', then we should scale it separately in _susc, _inf functions, not in _foi

Reply:
Perfect.